Extract.hs

```diff
@@ -0,0 +1,28 @@
+import System.IO
+import System.Environment
+import Data.List
+
+main = do
+    args <- getArgs
+    header <- return( args !! 0)
```

Comment:
```suggestion
    let header = head args
```
two things here, typically you use `head` instead of `!!`, but you also don't need to return the value to then bind it again, you can simply use `let`.

You had to bind to args above as args is in the IO monad, but once you've bound, you can just index into it without any monad.